clever/api_client.py

```diff
@@ -258,10 +258,6 @@ def __deserialize(self, data, klass):
                 return {k: self.__deserialize(v, sub_kls)
                         for k, v in iteritems(data)}
 
-            if klass == 'Event':
-                klass = eval('models.'+''.join(data['type'].title().split('.')))
-                return self.__deserialize(data,klass)
-
             # convert str to class
```

Comment:
What this was doing was deciphering the model type of the event in question (District for event district.created) and deserializing the event body as though it was an instance of that model type, instead of using the Event model.